README.md

```diff
@@ -91,6 +91,40 @@ $ mkdir -p ~/.terraform.d/plugins/
 $ # Note: If a static binary is required, use ./bin/build to create the executable
 $ go build -o ~/.terraform.d/plugins/terraform-provider-conjur main.go
 ```
+### Access from Terraform Registry
+If you wish to use the Terraform Provider Conjur from Terraform Registry
```

Comment:
Suggested change:

> If you wish to use the Terraform Provider Conjur from Terraform Registry

to

> To use the Conjur Terraform Provider from the Terraform Registry:

I would at least change Terraform Provider Conjur to Conjur Terraform Provider to match the product name.

Reply:
Changes made and pushed